lib/ops_preflight/ember_tasks.rb

```diff
@@ -0,0 +1,46 @@
+namespace :ember do
+  desc 'Links the ember-cli-deploy dotenv config'
+  task link_config: :environment do
+    in_directory './ember' do
+      queue %[echo "-----> Symlinking ember deploy config"]
+      queue! echo_cmd(%{rm -rf "./.env.deploy.production"})
+      queue! echo_cmd(%{ln -s "#{deploy_to}/#{shared_path}/config/env.deploy.production" "./.env.deploy.production"})
+    end
+  end
+
+  desc 'Installs ember-cli and bower'
+  task install: :environment do
+    in_directory './ember' do
+      queue! %[
+        ln -sf "#{deploy_to}/#{shared_path}/node_modules" "./node_modules"
+        ln -sf "#{deploy_to}/#{shared_path}/bower_components" "./bower_components"
+      ]
+      queue %[
+        echo "-----> Installing ember dependencies"
+        #{echo_cmd %[npm install bower ember-cli]} &&
+        #{echo_cmd %[npm install]} &&
```

Comment:
Yeah, I know that much. I guess you are installing bower and ember-cli by name first and then installing the deps for the app. :+1: